integration-tests/oidc/src/test/java/io/quarkus/it/keycloak/KeycloakXTestResourceLifecycleManager.java

```diff
@@ -29,7 +29,7 @@ public class KeycloakXTestResourceLifecycleManager implements QuarkusTestResourc
     private static String KEYCLOAK_SERVER_URL;
     private static final String KEYCLOAK_REALM = "quarkus";
     private static final String KEYCLOAK_SERVICE_CLIENT = "quarkus-service-app";
-    private static final String KEYCLOAK_VERSION = System.getProperty("keycloak.version");
+    private static final String KEYCLOAK_VERSION = System.getProperty("keycloak.version", "23.0.1");
```

Comment:
I'm not sure how this property was supposed to be passed before? @sberyozkin ?

Reply:
Without you cannot run the tests from the IDE. It super inconvenient when you need to debug dozens of tests. 

I can revert that change, but you can be sure I will redo the same next time I need to debug these tests.